ipc/src/logger.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
```

Comment:
it's just logger can be used or not in tests
generally it is just turned on when trying to figure out why tests are broken